CHARTER.md

```diff
@@ -0,0 +1,117 @@
+# Project Charter
```

Comment:
One thing I'm struggling is whether _Crossplane_ mentioned here entails everything under `crossplane` org or `crossplane/crossplane` repository. For example, I know that this document means providers hosted under `crossplane` may no longer be but it's hard to get that this is changing. If that shouldn't be part of the charter, maybe we can include changes to `GOVERNANCE.md` resulting from this change so that it's more clear to folks what's changing in practice.

Reply:
Yeah I considered capturing that here too, but as you mentioned I think GOVERNANCE.md is the better place. I believe the current thinking is:

* Core, runtime, tools (incl Terrajet) all live under `github.com/crossplane`. Basically things that aren't extensions.
* Extensions _can_ live under `github.com/crossplane-contrib` if they want to be under project governance.

The immediate practical implication being that we'd move all providers to contrib.

Reply:
nitpick: I'd suggest to make that change to `GOVERNANCE.md` in this PR to make this clear to folks.

Reply:
I kind of agree, but since this has been approved I'm going to fast follow instead. :)